client/app/components/dashboards/AutoHeightController.js

```diff
@@ -0,0 +1,109 @@
+
+import { includes, reduce, some } from 'lodash';
+
+const WIDGET_SELECTOR = '[data-widgetid="{0}"]';
+const WIDGET_CONTENT_SELECTOR = [
+  '.widget-header', // header
+  'visualization-renderer', // visualization
+  '.scrollbox .alert', // error state
+  '.spinner-container', // loading state
+  '.tile__bottom-control', // footer
+].join(',');
+const INTERVAL = 500;
```

Comment:
Seems ok to me. Lmk if you think it should be 50 like the original.

Reply:
I think it's fine. We can adjust it later if needed.